tests/sdk.spec.ts

```diff
@@ -70,64 +70,128 @@ describe('HuddlySDK', () => {
   });
 
   describe('#setupDeviceDiscoveryListeners', () => {
+    const dummySerial = '123456';
     const dummyGO = {
-      productId: 0x11
+      productId: 0x11,
+      serialNumber: dummySerial
     };
     const dummyIQ = {
-      productId: 0x21
+      productId: 0x21,
+      serialNumber: dummySerial
     };
     let huddlygoInitStub;
     let boxfishInitStub;
     let discoveryEmitter;
     let otherEmitter;
     let hidInterfaceStub;
+    let dummyTargetSerial;
+
+    function initSdk() {
```

Comment:
Would be good to Use ES16 function declaration standard since we do that throughout all classes. In this case it would be:
```
initSdk() => {

...
}